src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.container.js

```diff
@@ -72,6 +72,17 @@ export const EnrollmentPageDefault = () => {
         dispatch(deleteEnrollment({ enrollmentId }));
     };
 
+    const onViewAll = (stageId) => {
+        history.push(
+            `/enrollment/stageEvents?${urlArguments({ orgUnitId, programId, stageId })}`);
+    };
+
+    const onCreateNew = (stageId) => {
+        history.push(
+            `/enrollmentEventNew?${urlArguments({ orgUnitId, programId, teiId, enrollmentId, stageId })}`,
+        );
+    };
```

Comment:
For performance reasons, I would recommend that we make it a habit to wrap callbacks in `useCallback`.

Reply:
do you mean we should add `useCallback` both here and in the [Stage file](https://github.com/dhis2/capture-app/pull/1944/files#diff-aa98c87a1bdf7b1b522ffd413b516c5423e36342210531a0d0e054a6c0df8789R73)?

Reply:
Yes, that makes sense to me